Index: vendor/plugins/acts_as_versioned/lib/acts_as_versioned.rb
===================================================================
--- vendor/plugins/acts_as_versioned/lib/acts_as_versioned.rb	(revision 1235)
+++ vendor/plugins/acts_as_versioned/lib/acts_as_versioned.rb	(working copy)
@@ -443,7 +443,7 @@
               self.connection.add_column versioned_table_name, :updated_at, :timestamp
             end
             
-            self.connection.create_index versioned_table_name, versioned_foreign_key
+            self.connection.add_index versioned_table_name, versioned_foreign_key
           end
 
           # Rake migration task to drop the versioned table
